docs/sources/datasources/influxdb/configure-influxdb-data-source/_index.md

```diff
@@ -0,0 +1,269 @@
+---
+aliases:
+  - ../data-sources/influxdb/
+  - ../data-sources/influxdb/provision-influxdb/
+  - ../features/datasources/influxdb/
+  - provision-influxdb/
```

Comment:
As this is a new page, does it need the aliases?

Reply:
removed